packages/cli/src/handlers/dbt/apiClient.ts

```diff
@@ -26,23 +25,12 @@ export const lightdashApi = async <T extends ApiResponse['results']>({
     body,
 }: LightdashApiProps): Promise<T> => {
     const config = await getConfig();
-    if (!(config.context?.apiKey && config.context.serverUrl)) {
+    if (!(config.context?.apiToken && config.context.serverUrl)) {
```

Comment:
would this be backwards compatible, I guess this change will force users to login again, since they will not have an apiToken on their config. 

Can we add something like (config.context?.apiToken || config.context?.apiKey) ?